### ISSUE TITLE
O2 percentage does not show up if "AIR"

### ISSUE THREAD

**Issue by jamespet77** [Subsurface-divelog/subsurface#3551]
- [ ] Bug
- [x] Change request
- [ ] New feature request
- [ ] Discussion request

### Issue long description:
When updating details of a dive in the equipment tab the O2 cell is sanitized and zero'd out if the O2 percentage falls in the Air category.   I thought this was a bug until I looked onto the code and noticed that this was intentional.  I don't really understand the reasoning other than it looks like some possible libdivecomputer math issues.  I think it'd be good to see "air" or something in that cell indicating that air is being recognized.  

### Operating system:
Linux and OSX

### Subsurface version:
5.0.10 - affects all versions

### Steps to reproduce:
Edit dive entering Cylinder O2 percentage of Air.   20.8 - 21.0 
### Current behavior:

The O2% does not display anything.   It is sanitized out.   
### Expected behavior:

Show Air percentage or display "AIR" as it does on the graph.  
<img width="1085" alt="Screen Shot 2022-11-07 at 7 50 58 PM" src="https://user-images.githubusercontent.com/378474/200464230-d188ae29-0394-425a-a881-008d8f2b8c14.png">

**Comment by alexason** [comment]
I Agree, was confused by this as well as a new user to Subsurface and inputting in some dives. I had initially thought it hadn't pulled through the gas data from my Perdix but as you have identified was just defaulting to nothing as it was Air.
